spec.md

```diff
@@ -1192,6 +1345,56 @@ message NodeGetIdResponse {
   // CO in subsequent `ControllerPublishVolume`.
   // This is a REQUIRED field.
   string node_id = 1;
+
+
+  // The topological segment(s) the provisioned volume is accessible
```

Comment:
This and the next 25 lines could probably be deleted - they frame this field from the perspective of a volume, and not a node. The first sentence of this comment should be the line that starts with "The topological segment(s) this node belongs..."

Reply:
Yep, that was a copy paste I forgot to clean up. Will remove.